app/helpers/link_to_helper.rb

```diff
@@ -254,6 +254,20 @@ def unhappy_url(info_request = nil)
     end
   end
 
+  # Google Analytics
+
+  # if used, value must an integer
+  def analytics_event_code(category, action, label=nil, value=nil, label_is_script=false)
```

Comment:
An options hash would be much clearer here. This could also do with some [documentation](http://tomdoc.org/) about what the arguments do. I wouldn't know how to use this by only looking at the method signature.